sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -435,6 +435,7 @@ public static class SubStatusCodes {
         //SDK Codes(Client)
         // IMPORTANT - whenever possible use consistency substatus codes that .Net SDK also uses
         public static final int TRANSPORT_GENERATED_410 = 20001;
+        public static final int TRANSPORT_GENERATED_503 = 2003;
```

Comment:
What is the purpose of this change?

Reply:
Left-over - we have a bug in Java where we don't use the appropriate substatuscode for the 503 generated when series of 410 fail
        public static final int TRANSPORT_GENERATED_503 = 20003;

Reply:
:net uses (an dodcuments 20003) for this

Reply:
we use 20001 for both 410/20001 and 503/20001 - but no intend to change in this PR - reverted